specification/synapse/data-plane/Microsoft.Synapse/preview/2021-07-01-preview/artifacts.json

```diff
@@ -0,0 +1,5064 @@
+          {
+            "$ref": "#/parameters/KqlScriptName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "kqlScript",
+            "description": "KQL script",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/KqlScriptResource"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/KqlScriptResource"
+            }
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../resource-manager/Microsoft.Synapse/preview/2019-06-01-preview/workspace.json#/definitions/ErrorContract"
+            }
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-examples": {
+          "KqlScript_CreateOrUpdate": {
+            "$ref": "./examples/KqlScript_CreateOrUpdate.json"
+          }
+        }
+      },
+      "get": {
+        "tags": [
+          "kqlScripts"
+        ],
+        "operationId": "KqlScript_GetByName",
+        "description": "Get KQL script by name",
+        "parameters": [
+          {
+            "$ref": "#/parameters/KqlScriptName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/KqlScriptResource"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "../../../../common/v1/types.json#/definitions/ErrorContract"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "KqlScript_GetByName": {
+            "$ref": "./examples/KqlScript_GetByName.json"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "kqlScripts"
+        ],
+        "operationId": "KqlScript_DeleteByName",
+        "description": "Delete KQL script by name",
+        "parameters": [
+          {
+            "$ref": "#/parameters/KqlScriptName"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v1/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK"
+          },
+          "202": {
+            "description": "Accepted"
+          },
+          "204": {
+            "description": "No Content"
+          },
```

Comment:
A delete operation should return only `204` or only `202`.

Reply:
@epkalyanr I am not sure the real status of the deletion operation. Could you please confirm?